### ISSUE TITLE
New Tab Black - HELP!

### ISSUE THREAD

**Issue by mfrumento** [darktrojan/newtabtools#451]
I clicked on new tab last night and the whole thing is black except for the settings wheel.  I can go in but nothing will save.  All of my tabs are gone.  Please help!

**Comment by indivisible-irl** [comment]
+1 on this, only settings cog and the padlock visible in the corners. 
On FF 68.0b7 now but happening since or before the recent cert renewal fiasco I think so 67b somewhere.

I'm prob overdue a profile refresh anyway and have been thinking about going back to the release branch again now that 67 has been pushed out - this might be the nudge for me to finally get around to doing it. If it's not working there on a clean install I'll update.

**Comment by mfrumento** [comment]
New profile didn’t work for me.  I had to install the FF update to get it working.

 

Michelle Frumento, CPDT-KA, CNWI
Certified Professional Dog Trainer
K9 Nose Work® Certified Instructor 
Your Mannerly Mutt, Inc.
www.yourmannerlymutt.com <blocked::blocked::http://www.yourmannerlymutt.com/> 
610-348-4267

Shop Your Mannerly Mutt: www.yourmannerlymutt.com/store <blocked::blocked::http://www.yourmannerlymutt.com/store> 

Please"Like" Your Mannerly Mutt on Facebook :
https://www.facebook.com/yourmannerlymutt <blocked::blocked::https://www.facebook.com/yourmannerlymutt> 

youtube channel:  www.youtube.com/mfrumento <blocked::blocked::http://www.youtube.com/mfrumento> 

CONFIDENTIALITY NOTICE: This email and any attachment are intended only for the named recipient, are the property of Michelle Frumento, and shall not be used, disclosed or reproduced without expressed written consent of Michelle Frumento. If you are not the intended recipient and have received it in error, you are notified that dissemination, distribution, or copying of this email and/or its attachments is strictly prohibited. If you have received this transmittal in error, please immediately inform Michelle Frumento and delete the original message and any copy of it in your possession. Your cooperation is required and appreciated.

**Comment by indivisible-irl** [comment]
Yeah, a new profile in 68.0b didn't help (same blank page with only settings/lock).
Clean install of 67.0.2 (release channel) has fixed it. Addon installed via FF Sync ok though no configs were retained (only the addon itself) so unsure if incompatable with FF 68b or some other profile corruption issue.

**Comment by mfrumento** [comment]
Sent you an email yesterday, I would really appreciate any help!

 

Michelle Frumento, CPDT-KA, CNWI
Certified Professional Dog Trainer
K9 Nose Work® Certified Instructor 
Your Mannerly Mutt, Inc.
www.yourmannerlymutt.com <blocked::blocked::http://www.yourmannerlymutt.com/> 
610-348-4267

Shop Your Mannerly Mutt: www.yourmannerlymutt.com/store <blocked::blocked::http://www.yourmannerlymutt.com/store> 

Please"Like" Your Mannerly Mutt on Facebook :
https://www.facebook.com/yourmannerlymutt <blocked::blocked::https://www.facebook.com/yourmannerlymutt> 

youtube channel:  www.youtube.com/mfrumento <blocked::blocked::http://www.youtube.com/mfrumento> 

CONFIDENTIALITY NOTICE: This email and any attachment are intended only for the named recipient, are the property of Michelle Frumento, and shall not be used, disclosed or reproduced without expressed written consent of Michelle Frumento. If you are not the intended recipient and have received it in error, you are notified that dissemination, distribution, or copying of this email and/or its attachments is strictly prohibited. If you have received this transmittal in error, please immediately inform Michelle Frumento and delete the original message and any copy of it in your possession. Your cooperation is required and appreciated.
